### ISSUE TITLE
[TW-1887] scheduled urgency doesn't work

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1900]
_Christian Krause on 2017-01-19T15:00:14Z says:_ 

 I have added a task with a due date and a scheduled date:

{code}
$ TASKRC=~/.taskrc-testing task add schedule test scheduled:2017-01-20T12:00:00 due:2017-01-20T16:00:00 rc.dateformat:Y-M-DTH:N:S
{code}

I have added another task with just a scheduled date:

{code}
$ TASKRC=~/.taskrc-testing task add schedule test scheduled:2017-01-20T12:00:00 rc.dateformat:Y-M-DTH:N:S
{code}

I have set a negative urgency coefficient for scheduled tasks:

{code}
$ TASKRC=~/.taskrc-testing task config urgency.scheduled.coefficient -12
Are you sure you want to add 'urgency.scheduled.coefficient' with a value of '-12'? (yes/no) yes
TASKRC override: /home/umcdev/.taskrc-testing
Config file /home/umcdev/.taskrc-testing modified.
{code}

This urgency coefficient is not applied to either of these tasks:

{code}
$ TASKRC=~/.taskrc-testing task 6,7 information
TASKRC override: /home/umcdev/.taskrc-testing

Name          Value
ID            6
Description   schedule test
Status        Pending
Entered       2017-01-19 15:36:17 (18min)
Scheduled     2017-01-20 12:00:00
Due           2017-01-20 16:00:00
Last modified 2017-01-19 15:36:17 (18min)
Virtual tags  DUE MONTH PENDING SCHEDULED TOMORROW UNBLOCKED WEEK YEAR
UUID          991ee664-6512-4a7d-aca1-cc5d92587a1d
Urgency       8.341

    due  0.695 *   12 =   8.34
* -----
                         8.341

Name          Value
ID            7
Description   schedule test
Status        Pending
Entered       2017-01-19 15:36:49 (17min)
Scheduled     2017-01-20 12:00:00
Last modified 2017-01-19 15:36:49 (17min)
Virtual tags  PENDING SCHEDULED UNBLOCKED LATEST
UUID          2d476789-b3c6-4b7c-8897-5ba5fb090fe0
Urgency          0

{code}
* ---

The due coefficient, however, gets applied. I think, I also would like it more if the due coefficient gets applied only if the task is *not* scheduled, i.e. while a task is *SCHEDULED*, *DUE* should not apply. I would report this in a separate issue, but this makes only sense if the scheduled urgency coefficient works.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2017-01-19T15:00:14Z
Modified: 2017-01-20T20:35:01Z
```

**Comment by taskwarrior** [comment]
_Eric Hymowitz on 2017-01-19T18:23:49Z says:_ 

 I had trouble with this concept too, originally.

For the "Urgency" calculation, "Scheduled" does not mean "this task has a date scheduled in the future."  "Scheduled" means "this task has a date scheduled for it, and that date has now passed, so the task is ready to be completed."

You might want to try "wait:" instead of "scheduled:" .

**Comment by taskwarrior** [comment]
_Christian Krause on 2017-01-20T13:43:13Z says:_ 

 Thanks for the nudge into the *wait* direction, I had already switched to using it instead of scheduled. It's a bit unintuitive at first, but now I got it how scheduled is supposed to work.

**Comment by taskwarrior** [comment]
_Christian Krause on 2017-01-20T13:45:16Z says:_ 

 I guess, we can close this issue. I don't seem to be able to find out how, I don't have permission to do this, have I?

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2017-01-20T20:35:01Z says:_ 

 Closing issue - Christian has an answer.
